src/SIL.XForge.Scripture/locales.json

```diff
@@ -4,14 +4,14 @@
     "englishName": "English (US)",
     "tags": ["en", "en-US"],
     "production": true,
-    "helps": "en"
+    "helps": ""
```

Comment:
Docusaurus puts English at `/`, not `/en`, hence this change. Right now our site has an HTML file served at `/en` that does a redirect to `/`, which is why it's not currently broken. But we can simplify things by linking to the right place.